<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivingRoomDialogClass</class>
 <widget class="QWidget" name="LivingRoomDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1352</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1352</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="verticalWidget_a" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246,247,250);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_b">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="topWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="topLeftWidget" native="true">
                <widget class="QLabel" name="roomIDLabel">
                 <property name="geometry">
                  <rect>
                   <x>200</x>
                   <y>14</y>
                   <width>141</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(122,122,122);</string>
                 </property>
                 <property name="text">
                  <string>RoomID:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="anchorIDLabel">
                 <property name="geometry">
                  <rect>
                   <x>350</x>
                   <y>14</y>
                   <width>141</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(122,122,122);</string>
                 </property>
                 <property name="text">
                  <string>主播ID:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="logoGraphicsView">
                 <property name="geometry">
                  <rect>
                   <x>13</x>
                   <y>10</y>
                   <width>173</width>
                   <height>22</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-image: url(:/joy/logo);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
                <widget class="QLabel" name="nickLabel">
                 <property name="geometry">
                  <rect>
                   <x>500</x>
                   <y>14</y>
                   <width>181</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(122,122,122);</string>
                 </property>
                 <property name="text">
                  <string>昵称:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="roomNameLabel">
                 <property name="geometry">
                  <rect>
                   <x>690</x>
                   <y>14</y>
                   <width>241</width>
                   <height>17</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(122,122,122);</string>
                 </property>
                 <property name="text">
                  <string>房间名称:</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="topRightWidget" native="true">
                <property name="minimumSize">
                 <size>
                  <width>72</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>72</width>
                  <height>16777215</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>12</number>
                 </property>
                 <property name="leftMargin">
                  <number>5</number>
                 </property>
                 <property name="topMargin">
                  <number>6</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="minBtn">
                   <property name="minimumSize">
                    <size>
                     <width>15</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>15</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-image: url(:/joy/min_button);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="closeBtn">
                   <property name="minimumSize">
                    <size>
                     <width>15</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>15</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-image: url(:/joy/close_button);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="mainWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QWidget" name="leftViewWidget" native="true">
                <widget class="QWidget" name="anchorViewWidget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>416</width>
                   <height>624</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QWidget" name="linkViewWidget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>416</x>
                   <y>0</y>
                   <width>416</width>
                   <height>624</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QPushButton" name="beginLivingBtn">
                 <property name="geometry">
                  <rect>
                   <x>357</x>
                   <y>470</y>
                   <width>118</width>
                   <height>36</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(255, 255, 255, 127);</string>
                 </property>
                 <property name="text">
                  <string>开播</string>
                 </property>
                </widget>
                <widget class="QWidget" name="linkUserInfoWidget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>416</x>
                   <y>0</y>
                   <width>416</width>
                   <height>49</height>
                  </rect>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>416</width>
                   <height>49</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                 </property>
                 <widget class="QLabel" name="linkUserNickLabel">
                  <property name="geometry">
                   <rect>
                    <x>9</x>
                    <y>6</y>
                    <width>109</width>
                    <height>18</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>8</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>主播昵称</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="linkUserUIdLabel">
                  <property name="geometry">
                   <rect>
                    <x>9</x>
                    <y>26</y>
                    <width>120</width>
                    <height>18</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>8</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>UID:</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="linkBreakBtn">
                  <property name="geometry">
                   <rect>
                    <x>360</x>
                    <y>10</y>
                    <width>41</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(0, 0, 0, 25);</string>
                  </property>
                  <property name="text">
                   <string>断开</string>
                  </property>
                 </widget>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="centerListWidget" native="true">
                <property name="minimumSize">
                 <size>
                  <width>240</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>240</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="rightBeautyWidget" native="true">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>240</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>240</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="7" margin="11"/>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
